.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
You can add this to your .gitignore file in the future to prevent it from being pushed to github
https://www.pluralsight.com/guides/how-to-use-gitignore-file

Not at all mandatory, but nice to know